e2e-tests/playwright/specs/accessibility/channels/settings_dialog/notifications.spec.ts

```diff
@@ -0,0 +1,374 @@
+        await pw.toBeFocusedWithFocusVisible(settingsModal.notificationsTab);
+
+        // # Press Tab to move focus to Learn more link
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(notificationsSettings.learnMoreText);
+
+        // # Press Tab to move focus to Desktop and mobile notifications button
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(notificationsSettings.desktopAndMobileEditButton);
+
+        // # Press Tab to move focus to Desktop notification sounds button
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(notificationsSettings.desktopNotificationSoundEditButton);
+
+        // # Press Tab to move focus to Email notifications button
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(notificationsSettings.emailEditButton);
+
+        // # Press Tab to move focus to Keywords that trigger notifications button
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(notificationsSettings.keywordsTriggerNotificationsEditButton);
+
+        // # Press Tab to move focus to Keywords that get highlighted (without notifications) button
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(notificationsSettings.keywordsGetHighlightedEditButton);
+
+        // # Press Tab to move focus to Send a test notification button
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(notificationsSettings.testNotificationButton);
+
+        // # Press Tab to move focus to Troubleshooting docs button
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(notificationsSettings.troubleshootingDocsButton);
+    },
+);
+
+test(
+    'accessibility scan and aria-snapshot of Notifications settings panel',
+    {tag: ['@accessibility', '@settings', '@notification_settings', '@snapshots']},
+    async ({pw, axe}) => {
+        // # Create and sign in a new user
+        const {user, adminClient} = await pw.initSetup();
+        const clientConfig = await adminClient.getClientConfig();
+
+        // # Log in a user in new browser context
+        const {page, channelsPage} = await pw.testBrowser.login(user);
+        const globalHeader = channelsPage.globalHeader;
+        const settingsModal = channelsPage.settingsModal;
+
+        // # Visit a default channel page
+        await channelsPage.goto();
+        await channelsPage.toBeVisible();
+
+        // # Set focus to Settings button and press Enter
+        await globalHeader.settingsButton.focus();
+        await page.keyboard.press('Enter');
+
+        // * Settings modal should be visible
+        await expect(settingsModal.container).toBeVisible();
+
+        // * Verify aria snapshot of Notifications settings tab
+        await expect(settingsModal.notificationsSettings.container).toMatchAriaSnapshot(`
```

Comment:
Why are some of these snapshots inline while other are in separate files? Are there any guidelines when we should use inline snapshots or not?